nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -439,6 +452,12 @@
     private static final String RFC_1123 = "EEE, dd MMM yyyy HH:mm:ss 'GMT'";
     private static final DateTimeFormatter DATE_FORMAT = DateTimeFormat.forPattern(RFC_1123).withLocale(Locale.US).withZoneUTC();
 
+    /**
+     * The maximum size, in bytes, that the ETag cache should grow if it is enabled.
+     * The size here is chosen arbitrarily, as the documentation doesn't suggest a recommended value.
+     */
+    private static final int MAX_ETAG_CACHE_SIZE_BYTES = 1024 * 1024 * 10; // 10MiB
```

Comment:
Making this configurable with a property descriptor would be a good idea. I think two property descriptors, one for the numeric portion and the other for the unit would be very helpful to users.